baby-liminal-extension/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+name = "baby-liminal-extension"
+version = "0.1.0"
+authors = ["Cardinal Cryptography"]
+edition = "2021"
+homepage = "https://alephzero.org"
+repository = "https://github.com/aleph-zero-foundation/aleph-node"
+license = "Apache 2.0"
+publish = false
+
+[dependencies]
+ink = { git = "https://github.com/paritytech/ink", rev = "4655a8b4413cb50cbc38d1b7c173ad426ab06cde", default-features = false, optional = true }
+
+scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
+
+obce = { git = "https://github.com/727-Ventures/obce", branch = "liminal-v0.9.32", default-features = false }
```

Comment:
do we require some specific twists, so that we cannot use the main version?

Reply:
Yes, primarily the environment mocking capabilities are only possible because `liminal` has `ChargedWeight` exposed.

Reply:
oh, right... that's sad - do you think that if we give this testing up for now, we could use `main`? the two branches a bit desynchronized and I would prefer keeping up to the most recent

Reply:
Reviewing the `liminal-0.9.32` branch again, there is another issue with `ChargedWeight` being private - OBCE cannot provide the pre-charged amount of weight to the called method, since storing the `ChargedWeight` inside extension context requires us to name a (private) type.